### ISSUE TITLE
ontonotes coreference data

### ISSUE THREAD

**Issue by lovodkin93** [nyu-mll/jiant#1102]
Hello,
So going over the ontonotes data, I understood that the last column represents the coreference relations, as seen in the following screen shot:
![image](https://user-images.githubusercontent.com/57570615/86625643-14b81d00-bfce-11ea-9eb0-7bacc1cc7148.png)
as can be seen in the screenshot, two entities that are related get the same number in the last column.
Now, my question is - do these numbers have a special meaning, other than the fact the connected entities get the same ones? Or does every group of connected entitites get their index at random?
In other words, if for example I change all the "42" in all the data with a different number (that doesn't appear in the data at all) - will it affect anything?
Thanks!

**Comment by sleepinyourhat** [comment]
If you're looking at the raw data, I'm not sure we'll be able to help. You'd be better off contacting the creators of Ontonotes. 

If you're specifically focused on how jiant 1.3 loads the data, @iftenney or @zphang might be able to help.

**Comment by zphang** [comment]
*This is an automatically generated comment.*

As we update `jiant` to v2.x, `jiant` v1.x has been migrated to https://github.com/nyu-mll/jiant-v1-legacy. As such, we are closing all issues relating to `jiant` v1.x in this repository.

If this issue is still affecting you in `jiant` v1.x, please follow up at https://github.com/nyu-mll/jiant-v1-legacy/issues/1102.

If this issue is still affecting you in `jiant` v2.x, reopen this issue or create a new one.
